datadog_checks_base/datadog_checks/base/utils/subprocess_output.py

```diff
@@ -59,6 +59,8 @@ def get_subprocess_output(command, log, raise_on_empty_output=True, log_debug=Tr
         'get_subprocess_output returned (len(out): %s ; len(err): %s ; returncode: %s)', len(out), len(err), returncode
     )
 
+    log.debug('out: %s ; err: %s', out, err)
```

Comment:
Let's remove this

Reply:
Was about to comment the same thing :-)

```suggestion
```

Probably an oversight, if not then exposing stdout/stderr _may_ be okay, but should be done safely as we (probably?) don't want to leak possible secrets or sensitive info into logs. So it would need its own small discussion and PR.

Reply:
Can you share the reason why it should be removed? The issue could be figured out easier by support professionals if we had this output.